src/Microsoft.ML.Data/Training/ITrainerEstimator.cs

```diff
@@ -4,10 +4,18 @@
 
 namespace Microsoft.ML.Trainers
 {
+    /// <summary>
+    /// Interface for the Trainer Estimator.
+    /// </summary>
+    /// <typeparam name="TTransformer"></typeparam>
+    /// <typeparam name="TPredictor"></typeparam>
     public interface ITrainerEstimator<out TTransformer, out TPredictor> : IEstimator<TTransformer>
```

Comment:
Interesting! It's still named `TPredictor`, we have switched to `TModel` in out public facing API. If you could rename it here it would be great!